test/parseedn-test-data.el

```diff
@@ -243,6 +243,36 @@
                                              (:form . "123")
                                              (:value . 123)))))))))
 
+       "prefixed-map"
+       (a-list
+        :source "#:foo.bar{:baz 1 :other.ns.prefix/qux 2}"
```

Comment:
A few variations that you might want to include into the test suite:

* `#:foo.bar {` (i.e. space before the bracket)
* `#::{:bar 2}` (expands to the current ns)
* `#::foo{:bar 2}` (expands to an aliased ns)

Reply:
While the first variation makes total sense, second and third don't (to me). As far as I know, auto-resolved keywords are not supported in EDN. I omitted a part of that detail in my quote in the commit message, but Alex Miller said https://github.com/edn-format/edn/issues/78: 

> The Clojure edn reader was also updated to support the non-autoresolved parts of namespace map syntax **(edn doesn't do anything autoresolved like ::foo or #::foo{})**.

(emphasis mine). Which IMHO, makes total sense. EDN is not Clojure and doesn't have the concept of the current namespace. The following code (using Clojure 1.9.0) show that the syntax is not supported by the reference implementation in Clojure:

```
user=> (require '[clojure.edn :as edn])
nil
user=> (edn/read-string "#::{:bar 2}")
RuntimeException Invalid token: :  clojure.lang.Util.runtimeException (Util.java:221)

user=> (edn/read-string "#::foo{:bar 2}")
RuntimeException Namespaced map must specify a valid namespace: :foo  clojure.lang.EdnReader$NamespaceMapReader.invoke (EdnReader.java:495)

user=> (edn/read-string "{::foo/bar 2}")
RuntimeException Invalid token: ::foo/bar  clojure.lang.Util.runtimeException (Util.java:221)

user=> (edn/read-string "{::bar 2}")
RuntimeException Invalid token: ::bar  clojure.lang.Util.runtimeException (Util.java:221)

user=>
```

Reply:
Got it, thanks!

True, parseedn != parseclj.